plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -660,15 +691,19 @@ int flb_stackdriver_conf_destroy(struct flb_stackdriver *ctx)
     if (ctx->regex) {
         flb_regex_destroy(ctx->regex);
     }
-    
+
     if (ctx->project_id) {
         flb_sds_destroy(ctx->project_id);
     }
-    
+
     if (ctx->tag_prefix_k8s) {
         flb_sds_destroy(ctx->tag_prefix_k8s);
     }
 
```

Comment:
Don't you need to destroy `ctx->cloud_logging_base_url` too? Or is this only for the fields that aren't registered in the config map?

Reply:
Since `ctx->cloud_logging_base_url` is set from the config map, it will be destroyed here: https://github.com/fluent/fluent-bit/blob/68f1887f46547561213db2e1e89571b04d3b8594/src/flb_output.c#L96

(this is a bit confusing to trace, and I only found it while valgrinding when I had originally manually destroyed it here and got an invalid free at that line)